apps/devtools/app/scripts/styles/theme.ts

```diff
@@ -35,4 +35,23 @@ const colors = {
 export const theme = extendTheme({
   config,
   colors,
+  styles: {
+    global: {
+      '*': {
+        '&::-webkit-scrollbar': {
+          width: '1px',
```

Comment:
We might want to think a bit more about the accessibility with this one, but feel like this is some thing we can address when it comes down the road.